### PR TITLE
qa: don't expect "file exists" error for "mkdir -p"

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
"mkdir -p" doesn't raise any error when directory to be created already
exists. Also, use chmod command instead of mkdir command to set
permission mode on directory since mkdir command would have no effect
on directory's permission mode if directory already exists.

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by rishabh-d-dave** [comment]
Tests passed -

kernel: http://pulpito.front.sepia.ceph.com/rishabh-2021-03-05_09:03:34-fs-wip-rishabh-vr-reuse-AFAP-distro-basic-smithi/
FUSE: http://pulpito.front.sepia.ceph.com/rishabh-2021-03-05_06:37:14-fs-wip-rishabh-vr-reuse-AFAP-distro-basic-smithi/

File: qa/tasks/cephfs/fuse_mount.py [hunk 1]
```diff
@@ -57,16 +57,12 @@ def _mount(self, mntopts, check_status):
            self.client_config.get('valgrind') is not None:
             daemon_signal = 'term'
 
+        self.client_remote.run(args=f'mkdir -p -v {self.hostfs_mntpt}',
+                               timeout=60, stderr=StringIO())
```
Comment (batrick) [inline qa/tasks/cephfs/fuse_mount.py:61]: Is there a reason to do `stderr=StringIO()` if we're not going to look at it?

**Review by batrick — CHANGES_REQUESTED** [review]

File: qa/tasks/cephfs/fuse_mount.py (hunk 1 above)
Comment (rishabh-d-dave) [reply qa/tasks/cephfs/fuse_mount.py]: Yep, that can be removed as well!

Removed it in the current push! Thanks!

**Review by batrick — APPROVED** [review]

**Review by lxbsz — APPROVED** [review]
LGTM.

**Comment by batrick** [comment]
https://pulpito.ceph.com/pdonnell-2021-03-15_22:16:56-fs-wip-pdonnell-testing-20210315.182203-distro-basic-smithi/

- https://tracker.ceph.com/issues/49842
- https://tracker.ceph.com/issues/49466
- https://tracker.ceph.com/issues/49822
- https://tracker.ceph.com/issues/49240
- https://tracker.ceph.com/issues/48773
- https://tracker.ceph.com/issues/45434
- https://tracker.ceph.com/issues/49500
- https://tracker.ceph.com/issues/49843
- https://tracker.ceph.com/issues/49845
- https://tracker.ceph.com/issues/48805
- https://tracker.ceph.com/issues/49605

and failure caused by PR: https://github.com/ceph/ceph/pull/39969

**Pull request merged**